ert_shared/models/multiple_data_assimilation.py

```diff
@@ -67,6 +67,8 @@ def runSimulations(self, arguments):
             self._simulateAndPostProcess(run_context, arguments)
 
             EnkfSimulationRunner.runWorkflows(HookRuntime.PRE_UPDATE, ert=ERT.ert)
+            if is_first_iteration:
+                EnkfSimulationRunner.runWorkflows(HookRuntime.PRE_FIRST_UPDATE, ert=ERT.ert)
```

Comment:
Notice that this is not the same order as in the ES ;)

Reply:
Good point, will try refactoring this a bit, and add some testing on the order of the hooks.